docs/ref/wj.md

```diff
@@ -18,7 +18,7 @@ wj1[w; c; t; (q; (f0;c0); (f1;c1))]
 
 Where
 
--   `t` and `q` are simple tables to be joined (`q` should be sorted `` `sym`time `` with `` `p# `` on sym)
+-   `t` and `q` are simple tables to be joined (`q` should be sorted `` `sym`time `` with `` `p# `` on sym). Since 4.1t 2023.08.04 if `t` is the name of a table, it is updated in place.
```

Comment:
And table q?

Reply:
would give an error, as would trying to feed in anything else for 'q' that isnt a table

Reply:
OK.